src/libraries/Microsoft.PowerFx.Core/Utils/TextFormatUtils.cs

```diff
@@ -44,9 +44,10 @@ internal sealed class TextFormatUtils
         /// </summary>
         /// <param name="formatString">Raw input format string.</param>
         /// <param name="formatCulture">Current format culture.</param>
+        /// <param name="configurationLanguage">Configuration language.</param>
```

Comment:
>Configuration language. [](http://example.com/codeflow?start=48&length=23)

what does this mean?
ie, is it a default arg for something?

Reply:
yes, it is default configuration language. if we dont have language code in format string, get it from default language. i can change variable name.